src/real-time/index.js

```diff
@@ -0,0 +1,9 @@
+import createSocket from 'socket.io-client';
+
+const { REACT_APP_BACKEND_URL } = process.env;
+
+const socket = createSocket(REACT_APP_BACKEND_URL);
+
+const connect = (event, callback) => socket.on(event, data => callback(data));
+
+export default connect;
```

Comment:
add a line at end of page

Reply:
And follow proper naming conventions for the folder `real-time`

Reply:
> add a line at end of page

I already added a line but it appears it is not showing.

<img width="1214" alt="Screen Shot 2019-03-14 at 10 30 45 AM" src="https://user-images.githubusercontent.com/26969309/54346055-ab780200-4644-11e9-8945-c550bbca3348.png">
<img width="918" alt="Screen Shot 2019-03-14 at 10 31 35 AM" src="https://user-images.githubusercontent.com/26969309/54346057-ac109880-4644-11e9-9924-99987fc72657.png">